doc/user/layouts/partials/head.html

```diff
@@ -84,6 +84,22 @@
       container: "#docsearch",
     });
   });
+</script> -->
```

Comment:
Please just delete this instead of commenting it out—it's in the `git log` if we need it!